### PR TITLE
Add client swagger definition

### PR DESCRIPTION
**Pull request by kvigen**
Slightly different from the definitions we have for the documentation.
The client libraries have removed some of the features we don't think
are as useful and would just add clutter.

### CODE REVIEW

File: v1.2-client.yml [hunk 1]
```diff
@@ -0,0 +1,1797 @@
...
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/SchoolResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+      summary: Get only a specific school's information.
+  /schools/{id}/district:
```
Comment (beenakamath) [inline v1.2-client.yml]: add empty line above

File: v1.2-client.yml [hunk 2]
```diff
@@ -0,0 +1,1797 @@
...
+
+  studentcontacts.deleted:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: StudentContactObject
+
+  districts.updated:
```
Comment (beenakamath) [inline v1.2-client.yml]: what do these created/updated/deleted definitions do?

Comment (kvigen) [inline v1.2-client.yml]: they're used by the java code to implement polymorphism. check out https://github.com/Clever/clever-java/blob/a0105a0205baeb1f91a6350ef62448f2774efa96/override/JSON.java#L101-L136

File: v1.2-client.yml (hunk 1 above)
Comment (kvigen) [inline v1.2-client.yml]: https://github.com/Clever/swagger-api/pull/5/commits/478726e54c771390e5c36927e53d181819b08bbf

**Pull request merged**